nupm/install.nu

```diff
@@ -0,0 +1,84 @@
+use std log
+
+def nupm-home [] {
+    $env.NUPM_HOME? | default (
+        $env.XDG_DATA_HOME?
+        | default ($nu.home-path | path join ".local" "share")
```

Comment:
Can't we store this in $nu.default-config-dir?

Reply:
do you mean having `nupm-home` always return `$nu.default-config-dir`? :thinking: 

i'm not sure these are *config* files, so i chose the XDG location, but i can change the default location, however, i think we should let the user change the location with something like `NUPM_HOME` :yum:

Reply:
should be good in ccf0329 now :wink: